source/model.js

```diff
@@ -52,35 +68,35 @@ function WinnerChecker(field){
                 }
             }
             if (isWinnerFound) {
-                return currentWinCombination = winArray[winSegment];
+                return winArray[winSegment];
             }
         }
         return false;
     }
 }
 
-function Player(name, symbol, field){
-    let gameField = field;
+function Player(name, symbol){
+    this.gameField = singletonField.getInstance();
```

Comment:
Please replace all occurrences of the this.gameField field in Player and ComputerPlayer with singletonField.getInstance()